packages/gp2-components/src/templates/__tests__/OnboardingBackground.test.tsx

```diff
@@ -0,0 +1,36 @@
+import { gp2 } from '@asap-hub/fixtures';
+import { render, screen } from '@testing-library/react';
+import OnboardingBackground from '../OnboardingBackground';
+
+const { createUserResponse } = gp2;
+
+it('renders the page description', () => {
+  const defaultProps = {
```

Comment:
Maybe create the defaultProps once at the beginning of the file and then call it when it's necessary, instead of creating it 3 times